projects/schematics/src/migrations/4_0/removed-public-api-deprecations/removed-public-api-deprecation.ts

```diff
@@ -1,39 +1,60 @@
 import { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
 import { DeprecatedNode } from '../../../shared/utils/file-utils';
 import { removedPublicApiDeprecation } from '../../mechanism/removed-public-api-deprecations/removed-public-api-deprecation';
+import {
+  PRODUCT_VARIANT_COMPONENT,
+  PRODUCT_VARIANT_GUARD,
+  PRODUCT_VARIANTS_MODULE,
+  SPARTACUS_STOREFRONTLIB,
+  VARIANT_COLOR_SELECTOR_COMPONENT,
+  VARIANT_SIZE_SELECTOR_COMPONENT,
+  VARIANT_STYLE_ICONS_COMPONENT,
+  VARIANT_STYLE_SELECTOR_COMPONENT,
+} from '../../../shared/constants';
 
 export const REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
-  // uncomment product variants deprecation on 4.0 migration works (#11391)
-  // // projects/storefrontlib/src/cms-components/product/product-variants/product-variants.component.ts
-  // {
-  //   node: PRODUCT_VARIANT_COMPONENT,
-  //   importPath: SPARTACUS_STOREFRONTLIB,
-  //   comment: `'${PRODUCT_VARIANT_COMPONENT}' was moved to @spartacus/product/variants.`,
-  // },
-  // // projects/storefrontlib/src/cms-components/product/product-variants/variant-color-selector/variant-color-selector.component.ts
-  // {
-  //   node: VARIANT_COLOR_SELECTOR_COMPONENT,
-  //   importPath: SPARTACUS_STOREFRONTLIB,
-  //   comment: `'${VARIANT_COLOR_SELECTOR_COMPONENT}' was moved to @spartacus/product/variants.`,
-  // },
-  // // projects/storefrontlib/src/cms-components/product/product-variants/variant-size-selector/variant-size-selector.component.ts
-  // {
-  //   node: VARIANT_SIZE_SELECTOR_COMPONENT,
-  //   importPath: SPARTACUS_STOREFRONTLIB,
-  //   comment: `'${VARIANT_SIZE_SELECTOR_COMPONENT}' was moved to @spartacus/product/variants.`,
-  // },
-  // // projects/storefrontlib/src/cms-components/product/product-variants/variant-style-icons/variant-style-icons.component.ts
-  // {
-  //   node: VARIANT_STYLE_ICONS_COMPONENT,
-  //   importPath: SPARTACUS_STOREFRONTLIB,
-  //   comment: `'${VARIANT_STYLE_ICONS_COMPONENT}' was moved to @spartacus/product/variants.`,
-  // },
-  // // projects/storefrontlib/src/cms-components/product/product-variants/variant-style-selector/variant-style-selector.component.ts
-  // {
-  //   node: VARIANT_STYLE_SELECTOR_COMPONENT,
-  //   importPath: SPARTACUS_STOREFRONTLIB,
-  //   comment: `'${VARIANT_STYLE_SELECTOR_COMPONENT}' was moved to @spartacus/product/variants.`,
-  // },
+  // projects/storefrontlib/src/cms-components/storefinder/store-finder.module.ts
+  {
+    node: PRODUCT_VARIANTS_MODULE,
+    importPath: SPARTACUS_STOREFRONTLIB,
+    comment: `'${PRODUCT_VARIANTS_MODULE}' was moved to @spartacus/product/variants.`,
```

Comment:
Could we point in the comments here specific entry points eg. @spartacus/product/variants/root or @spartacus/product/variants/components.

Also we should have everything removed listed in our 4_0.md migration doc.